src/views/CampusSafety/views/LostAndFound/views/MissingItemEdit/index.tsx

```diff
@@ -127,7 +156,37 @@ const MissingItemFormEdit = () => {
       dateLost: new Date(formData.dateLost).toISOString() || DateTime.now().toISO(),
       forGuest: false,
     };
+    const originalFields = Object.keys(formData);
+    const newFields = Object.keys(formDataCopy);
+    let newActionNote = '';
+    for (let i = 0; i < originalFields.length; i++) {
+      if (
+        // @ts-ignore
+        JSON.stringify(formDataCopy[originalFields[i]]) != JSON.stringify(formData[newFields[i]])
+      ) {
+        // @ts-ignore
+        newActionNote +=
+          originalFields[i] +
+          ': OLD: ' +
+          // @ts-ignore
+          formDataCopy[originalFields[i]] +
+          ', NEW: ' +
+          // @ts-ignore
+          formData[newFields[i]] +
+          ' ';
+      }
+    }
     await lostAndFoundService.updateMissingItemReport(requestData, parseInt(itemid || ''));
+    const actionRequestData = {
+      missingID: itemid,
```

Comment:
```suggestion
      missingID: parseInt(itemid || ''),
```
This tells Typescript that this will always be a number, which lets you remove the ts-ignore on line 188.